specs/lib/gltfPipelineSpec.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+var gltfPipeline = require('../../lib/gltfPipeline');
+var readGltf = require('../../lib/readGltf');
+var writeBinaryGltf = require('../../lib/writeBinaryGltf');
+var Cesium = require('cesium');
+var defined = Cesium.defined;
+
+var gltfPath = './specs/data/boxTexturedUnoptimized/CesiumTexturedBoxTest.gltf';
+var glbPath = './specs/data/boxTexturedUnoptimized/CesiumTexturedBoxTest.glb';
+
+describe('gltfPipeline', function() {
+    it('optimizes a gltf file', function(done) {
+        readGltf(gltfPath, function(gltf) {
+            var stringCopy = JSON.stringify(gltf);
```

Comment:
Now that I've looked more carefully, we should just use the `clone` library like some other tests do. Look at `writeShaderSpec` for example.